lib/evaluation/evaluationClientWrapper.ts

```diff
@@ -614,14 +690,15 @@ export class EvaluationClientWrapper {
       }
     }
 
-    // Output submission is same as input presentation submission, it's just that if it doesn't match, we return Error.
+    // Validate if the submission satisfies the presentation definition
```

Comment:
This comment is now changed into something worse and even simply describes the name of the method with a few more words. Obviously the comment wasn't about the single next line but the logic in the next few lines.

Reply:
reverted